### PR TITLE
Fix Linux Python Installer issues

### PR DESCRIPTION
**Pull request by spham-amzn**
## What does this PR do?

Fixes the following installer issues:

## Linux
* Skewed desktop icon for linux
* pybind errors when loading modules that import any ctypes (ie math) causing the error: `_ctypes.cpython-39-x86_64-linux-gnu.so: undefined symbol: PyFloat_Type`

| Before | After |
| -------- | -------- |
| 1_o3de_icon_before  |  1_o3de_icon_after    |

##  General

* Remove install time python bootstrapping. Bootstrapping of python is done by o3de automatically, so running o3de (project manager) will bootstrap the python files with the current current user permissions
    
```

## How was this PR tested?
* Created installer on linux and installed

### CODE REVIEW

**Review by forkercat — APPROVED** [review]

**Comment by amzn-changml** [comment]
Note that I have another PR in flight for removing the usage of python as root, along with fixes for cmake untar for windows: https://github.com/o3de/o3de/pull/18396. 

Lemme know if we should just combine this all into one and I can merge into yours

File: cmake/Platform/Windows/Packaging/PostInstallSetup.wxs [hunk 1]
```diff
@@ -56,12 +55,6 @@
             Execute="deferred"
             Impersonate="no"/>
 
-        <CustomAction Id="ConfigurePython"
```
Comment (amzn-changml) [inline cmake/Platform/Windows/Packaging/PostInstallSetup.wxs:59]: If we're removing the python bootstrapping, would bundling cmake really be necessary? I believe it was only bundled to solve this specific case during the install, and then rely on the user's cmake?

If so, we could provide alternate instructions for using `winget`: https://winget.run/pkg/Kitware/CMake

**Review by amzn-changml — APPROVED** [review]
Looks good, just a question about the dependency implication for this

File: cmake/Platform/Windows/Packaging/PostInstallSetup.wxs (hunk 1 above)
Comment (spham-amzn) [reply cmake/Platform/Windows/Packaging/PostInstallSetup.wxs:59]: We not removing bootstrapping all together, we are removing the bootstrap process as part of the install script. The project manager will bootstrap as necessary and as the current user. Python, the venv, and all the 3rd party files are all downloaded for the current user so there is no need for root access for them in the first place

**Comment by spham-amzn** [comment]
Don't wait for this PR  I still need to do extensive testing before I merge this in

**Pull request merged**